Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apk add --update git make bash
 # Get dependencies
 
 WORKDIR /w
-COPY go.mod go.sum /w/
+COPY go.mod go.sum ./
```

Comment:
It's also copying from `/w` on line 24

Reply:
That should be fine - the WORKDIR is /w, right?